README.md

```diff
@@ -168,6 +168,24 @@ By default, `snyk-monitor` uses an emptyDir for temporary storage. If you prefer
 The PVC's name defaults to `snyk-monitor-pvc`. If you prefer to override this, then use the following value:
 * `pvc.name`
 
+## Configuring excluded namespaces ##
```

Comment:
I know it's confusing but this is the README of our YAML installation, the Helm one is under `snyk-monitor/README.md` 😄  Probably we should have only one copy of the README and only for Helm since the YAML way is falling out of favour/support from us...